resources/demoshop.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Swedbank Pay Resources Demoshop
+title: Swedbank Pay Resources
```

Comment:
The `title` of `demoshop.md` should be `Swedbank Pay Demoshop`, no? Did you mean to do this in the `index.md` file?